src/daomop/plant.py

```diff
@@ -19,11 +21,11 @@ def __init__(self, prefix, field, ccd, version, ext):
 
     @property
     def filename(self):
-        return os.path.basename(storage.get_cands_uri(self.field,
-                                                      ccd=self.ccd,
-                                                      version=self.version,
-                                                      prefix=self.prefix,
-                                                      ext="measure3.{}.match".format(self.ccd)))
+        return os.path.basename(src.daomop.storage.get_cands_uri(self.field,
```

Comment:
again the src shows up.

I think the plant.py script is likely defunct regardless.